src/api/spec/controllers/webui/user_controller_spec.rb

```diff
@@ -220,6 +220,46 @@
         expect(user.roles).to match_array old_global_role
       end
     end
+
+    context "LDAP mode" do
+      let(:old_realname) { user.realname }
+      let(:old_email) { user.email }
+      let(:http_request) {
+        post :save, params: { user: { login: user.login, realname: 'another real name', email: 'new_valid@email.es' } }
+      }
+
+      before do
+        stub_const('CONFIG', CONFIG.merge({ 'ldap_mode' => :on }))
+      end
+
+      describe "as an admin user" do
+        before do
+          login admin_user
+        end
+
+        it "is not allowed to change real names of users" do
+          expect{ http_request }.not_to change{ user.reload ; user.realname }.from(old_realname)
+        end
+
+        it "is not allowed to change email addresses of users" do
+          expect{ http_request }.not_to change{ user.reload ; user.realname }.from(old_realname)
```

Comment:
And btw: this test does not what it states, it says it checks the email address but in fact it checks for realname.
Also a user only has one email address (see email addresses in the header)